src/Components/UserSettingsTwoFactorAuthentication/Components/AppSecondFactorMethod.tsx

```diff
@@ -0,0 +1,69 @@
+import { BorderBox, Button, Flex, Link, Sans, Serif } from "@artsy/palette"
+import { BorderBoxProps } from "@artsy/palette/dist/elements/BorderBox/BorderBoxBase"
```

Comment:
```suggestion
import { BorderBox, BorderBoxProps, Button, Flex, Link, Sans, Serif } from "@artsy/palette"
```

Reply:
I was also surprised by this error message:

```
import BorderBoxProps

Module '"../../../../node_modules/@artsy/palette/dist"' has no exported member 'BorderBoxProps'.ts(2305)
```

Reply:
Yep, you're absolutely right. See https://github.com/artsy/palette/blob/master/packages/palette/src/elements/BorderBox/BorderBox.tsx. 

We'll  fix it later. 👍